React/wbooks/src/redux/bookDetail/actions.js

```diff
@@ -0,0 +1,34 @@
+import UserService from '../../Services/UserService';
+
+export const fetchActions = {
```

Comment:
This should be in the same file as the one from book list. It belongs to the same Model